docs/cli-reference/dfx-canister.mdx

```diff
@@ -1137,14 +1138,17 @@ You can specify the following options for the `dfx canister update-settings` com
 | Option                                    | Description                                                                                                                                                                                                                                                                                                                                                                              |
 |-------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `--add-controller <principal>`            | Add a principal to the list of controllers of the canister.                                                                                                                                                                                                                                                                                                                              |
+| `--add-log-viewer <principal>`            | Add a principal to the list of log viewers of the canister.                                                                                                                                                                                                                                                                                                                              |
```

Comment:
same comment as for `--set-log-viewer`

Reply:
Fixed at https://github.com/dfinity/sdk/pull/3921/commits/201e2b727e6f000da193c446a0d68d4914eb3b0c.